multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/helpers/LiveRouteParameterHelperTest.java

```diff
@@ -0,0 +1,67 @@
+package org.cloudfoundry.multiapps.controller.core.helpers;
+
+import org.cloudfoundry.multiapps.controller.core.model.SupportedParameters;
+import org.cloudfoundry.multiapps.controller.core.validators.parameters.RoutesValidator;
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Module;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class LiveRouteParameterHelperTest {
```

Comment:
I think Sonar does not want test classes to be public so you can remove it
Apply format